packages/sitecore-jss/src/layout/rest-layout-service.test.ts

```diff
@@ -107,20 +100,12 @@ describe('RestLayoutService', () => {
       .get(
         '/sitecore/api/layout/render/jss?item=%2Fhome&sc_apikey=0FBFF61E-267A-43E3-9252-B77E71CEE4BA&sc_site=supersite&sc_lang=da-DK&tracking=false'
       )
-      .reply(200, (_, requestBody) => ({
-        requestBody: requestBody,
-        data: { sitecore: { context: {}, route: { name: 'xxx' } } },
-        headers: {
```

Comment:
You are not testing response headers anymore, rest of unit tests as well

Reply:
@addy-pathania we are still missing `headers` configuration and verification in unit tests